frontend/components/product-list/sections/FilterableProductsSection/useFilteredRefinementList.tsx

```diff
@@ -10,9 +10,17 @@ export function useFilteredRefinementList(props: UseRefinementListProps) {
    const hitsCount = results?.nbHits ?? 0;
    const isAnyRefined = items.some((item) => item.isRefined);
 
-   const filteredItems = isAnyRefined
+   let filteredItems = isAnyRefined
       ? items
       : items.filter((item) => hitsCount !== item.count);
 
-   return { ...rest, items: filteredItems };
+   let emptyFacetState = false;
```

Comment:
Looking at this again, I don't think `emptyFacetState` actually tells us what we want to know here. The issue is about showing grayed out facets instead of no facets. But this variable tells us whether a facet has any visible options inside of it, which doesn't really help us address the issue.

I think we might want to change some code in the `FacetsAccordian` instead. If all of it's `FacetAccordianItem` children are hidden, then there should be some logic to show those children grayed out instead of hiding them.

Reply:
Thanks Mason! Working on it now.

Reply:
Resolved d58eee9cc93e7d1bb0ad724515e9eb8c8db7de24